src/http/nodegen/middleware/applicationMiddleware.ts

```diff
@@ -13,8 +13,8 @@ import express from 'express';
 import expressFormData from 'express-form-data';
 import morgan from 'morgan';
 import { tmpdir } from 'os';
-import packageJson from 'package.json';
 import requestIp from 'request-ip';
+import packageJson from '../../../../package.json';
```

Comment:
Shouldn't this be possible with '@/package.json' as well?

Reply:
I would love to add a root alias - I think that's a great idea.

specifically `@/package` would currently reference the src folder, but we can add `@root/` or something like that, as `@/<rootfile>` might be confusing

Reply:
never seen this in a ts project before - lets leave it off for now and Jan could you try this in 1 of your projects and see how it goes? btw: this is how most languages work, a configurable setting to define where imports can come from..

Reply:
@p-mcgowan have you worked with an API where there is a root path shortcut?

Reply:
sure can, in a composer.json you can define exactly those autoloads with PSR-4 ;)
Sure thing will try it out somewhere first and create a MR then

Reply:
Nope, have not used root alias, but it could have been useful a few times - for example when sequelize config and migrations live outside the src folder, or if a frontend is importing backend typings from a shared root.

I'm easy either way - I really like that the aliases allow for moving files around very easily, but it it's just for this one file then  although it's a bit uggo I don't see a problem with leaving the relative path in - I think it's relatively uncommon to import package.json - in fact I believe there will be env vars set containing the package.name already